### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by plessl** [quarto-dev/quarto-cli#3307] [Q&A]
A question to the maintainers: What is the release schedule for Quarto?

It appears that the 1.2 release should be imminent as all issues related to this milestone have been fixed for a while. Are the minor releases tied to a particular event or date?

**Comment by dragonstyle** [comment]
A release of 1.2 is imminent- we have been stabilizing the build and it appears to have stabilized...

We don't typically tie releases to a specific date, but instead we're shooting for releases every few months (though we may extend or shrink these releases based upon what we're seeing). I'd expect 1.2 release version and pre-release builds of 1.3 to appear shortly.

**Reply by mcanouil** [reply]
This was really "shortly" 😂

@cscheid announced it an hour-ish ago https://mastodon.social/@scheidegger/109343921644281842

**Reply by cscheid** [reply]
@dragonstyle was halfway through making the release when he answered that ;)
